aptos-move/framework/aptos-token/sources/property_map.move

```diff
@@ -62,15 +62,15 @@ module aptos_token::property_map {
         values: vector<vector<u8>>,
         types: vector<String>
```

Comment:
I don't quite understand the signature of this `new` function since we expose the public api to construct a `PropertyValue` already so users could just pass in `keys` and `property_values` instead of `keys`, `types` and `values`. I think it breaks the encapsulation rule partially.

Similar feeling for `update_property_map`. If I want to call this functions with a value, I should be able to first call `create_property_value` and then use `update_property_map` but now I have to somehow get the `type` info of the value first but this module doesn't provide a way to do that. So for normal users, `update_property_map` seems not that useful.

Reply:
The new method was introduced earlier when we don't have the from_bcs function to actually create property value directly. 
we now can add a new public function for creating property_map with key and property_value directly